admin-dev/themes/new-theme/scss/pages/_translation_page.scss

```
@@ -1,5 +1,162 @@
+$color-error: #c05c67 !default;
+$color-active: #25b9d7 !default;
+
 .export-translations {
   .export-translations-child {
     padding-left: 50px;
   }
 }
+
+.breadcrumb {
+  small {
+    color: $gray-medium;
+  }
+}
+
+.total-remaining-translations {
+  color: $color-error;
+}
+
+.search-translation {
+  margin-bottom: 1.25rem;
+
+  .alerts {
+    margin-top: 1rem;
+  }
+
+  .col-offset {
+    margin-left: 1rem;
+  }
+
+  .hide {
+    display: none;
+  }
+
+  .summary {
+    font-size: 0.875rem;
+    color: #363a41;
+  }
+
+  .search {
+    width: 100%;
+    margin-top: 1px;
+  }
+
+  .search-button {
+    margin-right: 0.625rem;
+  }
+
+  .alert {
+    float: left;
+    width: 42.63rem;
+    height: 3.438rem;
+    margin-top: 1.875rem;
+
+    i,
+    span {
+      float: left;
+    }
+
+    span {
+      margin-left: 2.5rem;
+    }
+  }
+}
+
+.translation-domains {
+  float: left;
+  min-height: 50rem;
+  background-color: #fff;
+
+  h1 {
+    width: auto;
+    height: 1.375rem;
+    margin-top: 0.9375rem;
+    object-fit: contain;
+    font-size: $h1-font-size;
+    font-style: normal;
+    font-weight: 600;
+    font-stretch: normal;
+    color: #363a41;
```

```suggestion
    color: $gray-800;
```

Please do this for every colors, refer to `prestashop-ui-kit/scss/_variables.scss` for existing colors, overwise add a color to the config file of the PS Project